test/fixtures/v5/continue/uturn_exit.json

```diff
@@ -0,0 +1,22 @@
+{
+    "step": {
+        "maneuver": {
+            "type": "continue",
+            "modifier": "uturn"
+        },
+        "name": "Way Name",
+        "exits": "4A,4B"
+    },
+    "instructions": {
+        "de": "180°-Wendung auf Way Name",
+        "en": "Make a U-turn onto Way Name",
```

Comment:
Hm, thinking about situations for which a u-turn taking an exit occurs. As it's a complex maneuver for the driver do we really not want to add the exits information in addition here?